server/src/frameworks/Truffle/TruffleProject.ts

```diff
@@ -60,6 +60,18 @@ export class TruffleProject extends Project {
 
     this.globalNodeModulesPath = execSync("npm root --quiet -g").toString();
 
+    try {
+      // Ensure truffle is installed either globally or locally
+      require.resolve("truffle", {
```

Comment:
This check is breaking the truffle tests. Truffle isn't found locally or globally, so it throws.